### ISSUE TITLE
Test sets don't work well.  Does the test set data need the same preprocessing as the training set, or is the test data read directly from image_raw?

### ISSUE THREAD

**Issue by yanfangHao** [bowang-lab/U-Mamba#43]
Test sets don't work well. Does the test set data need the same preprocessing as the training set, or is the test data read directly from image_raw?
Answers would be appreciated, thank you very much!
测试效果不好。测试集数据是否需要进行与训练集相同的预处理，还是测试数据直接从 image_raw 读取？
希望能给出答案，非常感谢！

**Comment by yanfangHao** [comment]
I trained and tested on my own dataset
